parser/parser_test.go

```diff
@@ -828,7 +828,7 @@ func (s *testParserSuite) TestBuiltin(c *C) {
 
 		// for date, day, weekday
 		{"SELECT CURRENT_DATE, CURRENT_DATE(), CURDATE()", true},
-		{"SELECT CURRENT_DATE, CURRENT_DATE(), CURDATE(1)", true},
+		{"SELECT CURRENT_DATE, CURRENT_DATE(), CURDATE(1)", false},
```

Comment:
Why this is false case?

Reply:
@shenli  it seems that MySQL doesn't support `CURDATE` with one parameter. see: https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_curdate